{"error": "ERREUR: Entity with name 'IfcPipe' not found in schema 'IFC2X3'\nTraceback (most recent call last):\n  File \"C:\\Users\\wiemb\\Desktop\\flask-ifc-clashdetection\\app\\routes.py\", line 80, in process_clash_detection\n    detector.add_model(path)\n  File \"C:\\Users\\wiemb\\Desktop\\flask-ifc-clashdetection\\app\\clash_detection.py\", line 87, in add_model\n    elements_of_type = model.by_type(elem_type)\n  File \"C:\\Users\\wiemb\\Desktop\\flask-ifc-clashdetection\\venv\\lib\\site-packages\\ifcopenshell\\file.py\", line 532, in by_type\n    return [entity_instance(e, self) for e in self.wrapped_data.by_type(type)]\n  File \"C:\\Users\\wiemb\\Desktop\\flask-ifc-clashdetection\\venv\\lib\\site-packages\\ifcopenshell\\ifcopenshell_wrapper.py\", line 5750, in by_type\n    return _ifcopenshell_wrapper.file_by_type(self, *args)\nRuntimeError: Entity with name 'IfcPipe' not found in schema 'IFC2X3'\n", "status": "failed"}